Shared/Extensions/URLExtensions.swift

```diff
@@ -389,6 +389,14 @@ extension URL {
         }
         return nil
     }
+    
+    public var isMediaSiteURL: Bool {
+        // Don't need to include Github as it does a page load instead of XHR load.
+        guard let domain = self.baseDomain else {
+            return true
+        }
+        return ["youtube", "vimeo", "twitch", "twitter", "reddit"].contains(where: domain.contains)
```

Comment:
Does Vimeo, Twitter and Reddit all do this??? 😱

Reply:
Also shouldn't we only be doing this for Mobile YouTube/Twitch (m.youtube.com)

Reply:
@kylehickinson yeah all of them have the same issue.

Reply:
@kylehickinson for desktop too they behave same.

Reply:
Hmm, I wonder if Chromium sees these changes and handles them properly and its just WKWebView that doesn't

Reply:
I think wkwebview should be aware of this too, they are just not providing a callback for the same. Apart from url change kvo